### ISSUE TITLE
Merge `crouton_map` and `show_map="1"` features into one.

### ISSUE THREAD

**Issue by dgershman** [bmlt-enabled/crouton#303]
These two capabilities were developed at two different points in time.  They could certainly become one now.

**Comment by otrok7** [comment]
No, the behavior is very different, fitting to different use-cases.  [bmlt_tabs], with or without "show_map" gets all meetings at startup, and the map is basically another filter for which meetings are currently "in-view".  [crouton_map] makes multiple BMLT queries, base primarily on geographic location.  New location, new query.  More suitable for larger zones, etc.
